rules/manager_test.go

```diff
@@ -651,3 +652,86 @@ func TestUpdate(t *testing.T) {
 		}
 	}
 }
+
+func TestTemplateParsing(t *testing.T) {
+	// This is a dummy storage, just for the QueryFunc.
+	suite, err := promql.NewTest(t, "")
+	testutil.Ok(t, err)
+	queryFunc := EngineQueryFunc(suite.QueryEngine(), suite.Storage())
+
+	tests := []struct {
+		ruleString string
+		shouldPass bool
+	}{
+		{
+			ruleString: "" +
```

Comment:
Can we use:
`
- name: example
  rules:
  - alert: InstanceDown
    expr: up == 0
    for: 5m
    labels:
      severity: "page"
    annotations:
      summary: "Instance {{ $labels.instance }} down"

`

In golang, `string with whatever` is considered a string (i.e, use ` at the beginning and end) https://stackoverflow.com/questions/7933460/how-do-you-write-multiline-strings-in-go